src/backend/tokens.nim

```diff
@@ -9,14 +9,3 @@ proc getTokens*(chainId: int): RpcResponse[JsonNode] {.raises: [Exception].} =
 
 proc getBalances*(chainId: int, accounts: seq[string], tokens: seq[string]): RpcResponse[JsonNode] {.raises: [Exception].} =
   return callPrivateRPC("wallet_getTokensBalancesForChainIDs", %* [@[chainId], accounts, tokens])
-
-proc getCustomTokens*(): RpcResponse[JsonNode] {.raises: [Exception].} =
-  return callPrivateRPC("wallet_getCustomTokens", %* [])
-
-proc addCustomToken*(address: string, name: string, symbol: string, decimals: int, color: string) {.raises: [Exception].} =
```

Comment:
nvm, found them elsewhere

Reply:
@alaibe sorry because of a late reply... since we had `tokens.nim` and `custom_token.nim` in the `backend` I removed all api calls from `tokens` which points to `status-go` endpoints containing "custom" and that way avoid duplicated procs pointing to the same endpoint.